codegen/src/main.rs

```diff
@@ -65,26 +47,22 @@ fn main() -> anyhow::Result<()> {
     let verbose = matches.is_present("verbose");
 
     if stdout && output_path_glob.is_none() {
+        // TODO: What if the glob matches multiple files?
```

Comment:
This is a hangover from before I was using globs. I didn't really end up using this feature, it was easier to write output to a file to inspect it. Maybe it's useful to keep, but probably it would be better if I move this check until after the list of files has been generated and check that there's only one.

Reply:
Oops, I forgot to ask you about this and remove the todo again before having it merged. Sounds like you have some plans to proceed and clean it up though :)